.credo.exs

```diff
@@ -104,7 +104,7 @@
           {Credo.Check.Readability.ParenthesesInCondition, []},
           {Credo.Check.Readability.ParenthesesOnZeroArityDefs, []},
           {Credo.Check.Readability.PipeIntoAnonymousFunctions, []},
-          {Credo.Check.Readability.PredicateFunctionNames, []},
+          {Credo.Check.Readability.PredicateFunctionNames, [exit_status: 0]},
```

Comment:
TBH I don't know what we want to do with this but personally I don't think functions like `is_supported` `is_mandatory` etc are bad. Credo suggests using `supported?` and `mandatory?` which is imo less readable. See https://hexdocs.pm/credo/Credo.Check.Readability.PredicateFunctionNames.html